bin/deploy.sh

```diff
@@ -4,6 +4,7 @@ set -e
 export INGRESS_INTERNAL_ANNOTATIONS=$HOF_CONFIG/ingress-internal-annotations.yaml
 export INGRESS_EXTERNAL_ANNOTATIONS=$HOF_CONFIG/ingress-external-annotations.yaml
 export CONFIGMAP_VALUES=$HOF_CONFIG/configmap-values.yaml
+export CONFIGMAP1_VALUES=$HOF_CONFIG/cacert-store.yaml
```

Comment:
Could have a more descriptive name unless you wanted it to go a bit incognito

Reply:
certchain-store ?

Reply:
`CERTCHAIN_STORE` sounds fine to me

Reply:
Its going to be interesting when we have to define the cert we use by environment.